src/middlewared/middlewared/plugins/kubernetes_linux/update.py

```diff
@@ -36,6 +37,7 @@ class Config:
 
     ENTRY = Dict(
         'kubernetes_entry',
+        Bool('servicelb', required=True),
```

Comment:
Can we please add some documentation about it as well below in `do_update` doc string ?

Reply:
Thats a fair point, it's not a self-explainatory variable for people that are not into k3s 👍

Reply:
```suggestion
        # Allows for easy disabling of the k3s integrated Service LoadBalancer called "Klipper"
        Bool('servicelb', required=True),
```